single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DiagramLabelProvider.java

```diff
@@ -90,4 +92,8 @@ public LabelPosition getPosition() {
     default Optional<BusInfo> getBusInfo(BusNode node) {
         return Optional.empty();
     }
+
+    default Map<String, Side> getBusInfoSides(VoltageLevelGraph graph) {
+        return Collections.emptyMap();
```

Comment:
In fact I thought of doing directly like:
`graph.getNodeBuses().forEach(busNode -> getBusInfo(busNode).ifPresent(busInfo -> result.put(busNode.getId(), busInfo.getAnchor())));`